examples/chat/proto/chat.proto

```diff
@@ -21,47 +21,51 @@ package oak.examples.chat;
 import "google/protobuf/empty.proto";
 
 message CreateRoomRequest {
-  // Name of the room; this is only used for UI purposes (and so does not need
-  // to be unique).
+  // Name of the room; this is only used for UI purposes (and so does not need to be unique).
```

Comment:
I couldn't immediately see where the room name is now used &ndash; does it get lost?

Reply:
I guess it does get lost now. I don't see a strong reason for it over `room_id` now though.

We could (in descending order of my personal preference):

- remove it entirely
- add back proper support for it, adding a new method `GetRoomInfo` or similar that returns the name of the room given the `room_id` (not sure how to actually use it on the client though), and possibly a new type of message to change the room name

@daviddrysdale WDYT?

Reply:
The room name was aimed at giving humans a way to distinguish between rooms; "NKsceNlg69UbcvryfzmFGnMv9qnZ0DYh6u6gJxujnPPxvHsxMehoD368sumKawVaq9WaSkzrcStoNYLvVNdzhA==" isn't very friendly but "Oak Discussion" is more comprehensible (albeit spoofable).

So removing the room name feels like we're dropping functionality from what is already a fairly limited demo.  I was kind of expecting that the chat app would get extended (e.g. with an actual UI) rather than stripped down, so the latter option makes more sense to me.

Reply:
At this point the room name (and also the user handle) are effectively just part of the payload, it does not seem to matter much whether they are separate fields or just have the client format messages as "user_handle@room > text". The advantage of the current version is that we don't need separate types for `SentMessage` and `ReceivedMessage`, each user just sends a `Message`, and it is delivered as it is to the other users. Even the user handle seems a bit confusing, since as you say it does not actually correspond to anything (and I would be in favour of removing it at some point, in favour of something label-based that can be trusted, if we can come up with it -- perhaps integrity labels could play a role here).